components/n-ui/header/js/promoHandler.js

```diff
@@ -34,6 +34,10 @@ function isSignupForm () {
 		|| window.location.pathname === '/thank-you';
 }
 
+function isMyAccount () {
+	return document.documentElement.dataset.nextApp === 'control-centre';
```

Comment:
Just in case, it's probably worth ensuring `document.documentElement.dataset` exists before querying `nextApp` on it.

Reply:
done, thanks